frontend/src/app/modules/shared/components/dateTime/dateTime.component.ts

```diff
@@ -0,0 +1,49 @@
+/********************************************************************************
+ * Copyright (c) 2022, 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ * Copyright (c) 2022, 2023 ZF Friedrichshafen AG
+ * Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+import { Component, Input, Self } from '@angular/core';
+import { NgControl } from '@angular/forms';
+import { BaseInputComponent } from '@shared/abstraction/baseInput/baseInput.component';
+import { StaticIdService } from '@shared/service/staticId.service';
+
+type DateString = `${string}${string}${string}${string}-${string}${string}-${string}${string}`;
+type TimeString = `${string}${string}:${string}${string}`;
+export type DateTimeString = `${DateString}T${TimeString}`;
+
+@Component({
+  selector: 'app-dateTime',
```

Comment:
```suggestion
  selector: 'app-date-time',
```
to keep it consistent with other components' selector name convention?